### DISCUSSION TITLE
how can i send image to server ?

### DISCUSSION THREAD

**Discussion by ThorPham** [dropzone-ui/dropzone-ui-react#324] [Q&A]
Thank you for great library. I have 2 question.
first , I am use  InputButton and don't use any preview image component. How to automatic send image to server.
second, i want to use image data to render for other component. I see props "imageUrl" but it only get if i click view whole image. So i must convert from blob image use URL.createObjectURL(item.file) . Can i get directly image data ?
Thank you so much

**Comment by JinSSJ3** [comment]
Hey @ThorPham! thanks for your words.

We are curently working on a new version of 💻 @dropzone-ui/react and we are adding some new features like exposing the uploading function 📁 and also provide the upload feature for ⏺️ InputButton component, so in this new version your first question must be covered.
About the other question, wich format do you want to get the image data 🖼️? (e.g. base64, ObjectUrl). Before we finish the missing tasks we could add this feature to our backlog.

By the end of the next Tuesday (April 12nd) we'll have already deployed a new version. 
We will notify you when the next version is ready, so you can enjoy with these new features and others.
Best regards.
